.github/workflows/ci.yml

```
@@ -71,8 +73,8 @@ jobs:
 
       - uses: actions/setup-node@v3
         with:
-          node-version: "12.x"
+          node-version: "16.x"
 
-      - run: npm install
+      - run: yarn
```

```suggestion
      - run: yarn install
```

It does exactly the same, but it is more descriptive.